docs/installguide/install_all.rst

```diff
@@ -31,16 +31,13 @@ or use the command-line ``kalite`` program, which in typical installations can b
 Mac OS X
 ========
 
-.. warning:: The latest OS X version (EL Capitan) is not yet supported.
-
 Installation
 ____________
 
-#. Download the KA Lite `OSX installer <https://learningequality.org/r/osx-installer-0-15>`_.
-#. After the download is complete, double click the .dmg file.
-#. Click the ``Agree`` button to accept the LICENSE agreement.
-#. On the .dmg window, drag the ``KA-Lite Monitor`` app into the ``Applications`` folder.
-#. Launch ``KA-Lite Monitor`` from your ``Applications`` folder.
+#. Download the KA Lite `OSX installer <https://learningequality.org/r/osx-installer-0-16>`_.
+#. After the download is complete, double click the .pkg file.
+#. Click the ``Agree`` button to accept the LICENSE agreement and follow the installer instructions.
+#. Launch ``KA-Lite app`` from your ``Applications`` folder.
 #. On first load, it will check your current environment and show the Preferences dialog.
 #. Input your preferred administrator username and password, then click the ``Apply`` button in ``KA-Lite Preferences`` dialog.
 #. You will be prompted that initial setup will take a few minutes, click the ``OK`` button and wait for the notification that KA-Lite has been setup and can now be started.

```

Comment:
The KA Lite app is automatically launched during installation and we no longer ask for the admin account.  Please remove the steps from "Click the Agree button..." until "You will be prompted...".

Let's replace those 4 steps with these:
* Click on the Continue button to allow the installer program to check for pre-installation requirements.
* Follow the prompts in the installer dialog to install KA Lite.
* The "KA Lite app" will be launched automatically during installation, and should show notifications and a menu bar icon.
* When the installation is done, you will be notified that "KA Lite is running...".  The installer should also show the "Summary" page with instructions to start using KA Lite.
* To start using KA Lite, click on the menu bar icon and select "Open in Browser".